pyrene-graphs/jest.config.js

```diff
@@ -168,6 +168,8 @@ module.exports = {
   transform: {
     '^.+\\.js$': 'babel-jest',
     '^.+\\.jsx?$': 'babel-jest',
+    '^.+\\.ts$': 'babel-jest',
+    '^.+\\.tsx?$': 'babel-jest',
```

Comment:
I think the second line already covers the first one (`x?` -> x is optional and it matches both `.ts` and `.tsx`). Same for the js part.

Reply:
I totally agree! lol ! Thanks for the sharp eye :) Let me clean this up!